README.md

```diff
@@ -112,84 +125,141 @@ The content of your document is retrieved.
 *   [decodeDocument](#decodedocument)
     *   [Parameters](#parameters-2)
 *   [IEncryptionResults](#iencryptionresults)
-*   [encryptString](#encryptstring)
-    *   [Parameters](#parameters-3)
-*   [decryptString](#decryptstring)
-    *   [Parameters](#parameters-4)
+    *   [encryptString](#encryptstring)
+        *   [Parameters](#parameters-3)
+    *   [decryptString](#decryptstring)
+        *   [Parameters](#parameters-4)
 
 ### ENCRYPTION\_PARAMETERS
 
-Default options from responses
-[here](https://crypto.stackexchange.com/questions/26783/ciphertext-and-tag-size-and-iv-transmission-with-aes-in-gcm-mode/26787)
+To learn more about the default options from responses, see this [StackExchange post](https://crypto.stackexchange.com/questions/26783/ciphertext-and-tag-size-and-iv-transmission-with-aes-in-gcm-mode/26787).
 
 #### algorithm
 
+The algorithm is Galois/Counter Mode (GCM). It aims for authenticated encryption with associated data; and its specialization GMAC, for generating a
+message authentication code (MAC) on data that is not encrypted.
+
+GCM and GMAC are modes of operation for an underlying approved symmetric key block cipher. 
+
 #### keyLength
 
+The key size should be at least 128 bits. See this [NIST publication](http://csrc.nist.gov/publications/nistpubs/800-38D/SP-800-38D.pdf) to learn more.
```

Comment:
The default key size we use is 256

see here https://github.com/Open-Attestation/oa-encryption/blob/d44b38fc8ae9268907cd1ced6fb02250aab11439/src/index.ts#L7-L13

Reply:
updated

Reply:
noted, updated again based on the code